app/customers.py

```diff
@@ -0,0 +1,28 @@
+import math
+
+
+class Customer:
+    def __init__(self, user_info, fuel_price):
+        self.name = user_info["name"]
+        self.product_cart = user_info["product_cart"]
+        self.location = user_info["location"]
+        self.money = user_info["money"]
+        self.car = user_info["car"]
+        self.fuel_price = fuel_price
+
+    def fuel_costs(self, shop):
+        distance = math.dist(shop.location, self.location)
+        return self.fuel_price * distance * self.car["fuel_consumption"] / 100
+
+    def pay_for_trip(self, amount):
+        self.money -= amount
+        return self.money
```

Comment:
You don't need to return self.money. You can just use customer.money in main/$20